app/frontend/src/components/ImageInput/ImageInput.test.tsx

```diff
@@ -3,20 +3,28 @@ import userEvent from "@testing-library/user-event";
 import {
   CANCEL_UPLOAD,
   CONFIRM_UPLOAD,
+  COULDNT_READ_FILE,
   getAvatarLabel,
-  INVALID_FILE,
   SELECT_AN_IMAGE,
   UPLOAD_PENDING_ERROR,
 } from "components/constants";
 import { SUBMIT } from "features/constants";
+import { InitiateMediaUploadRes } from "proto/api_pb";
 import { useForm } from "react-hook-form";
 import { service } from "service";
+import client from "service/client";
+import {
+  IMAGE_TOO_LARGE,
+  INTERNAL_ERROR,
+  SERVER_ERROR,
+} from "service/constants";
 import wrapper from "test/hookWrapper";
-import { MockedService } from "test/utils";
+import { rest, server } from "test/restMock";
+import { assertErrorAlert, mockConsoleError, MockedService } from "test/utils";
 
 import ImageInput from "./ImageInput";
 
-const uploadFileMock = service.api.uploadFile as MockedService<
+let uploadFileMock = service.api.uploadFile as MockedService<
```

Comment:
I don't think we need to make this a `const` here since I can't see this being re-assigned from the changed code?